lib/utils/remote-build.ts

```diff
@@ -419,7 +419,7 @@ async function getRemoteBuildStream(
 	if (build.opts.headless) {
 		stream = buildRequest;
 	} else {
-		stream = buildRequest.pipe(JSONStream.parse('*'));
+		stream = buildRequest.pipe(JSONStream.parse('*')) as NodeJS.ReadStream;
```

Comment:
I don't love this extra casts, but the types from request are a bit weird to start with

Reply:
@pipex I also don't link this extra one, the other one already existed and is now a little bit better (as a ReadStream is slightly more conservative than the ReadWriteStream) but I don't think we have many other options